libcxx/include/__tree

```diff
@@ -1109,15 +1107,89 @@ public:
 
   // FIXME: Make this function const qualified. Unfortunately doing so
   // breaks existing code which uses non-const callable comparators.
+
+  // Find place to insert if __v doesn't exist
+  // Set __parent to parent of null leaf
+  // Return reference to null leaf
+  // If __v exists, set parent to node of __v and return reference to node of __v
```

Comment:
This needs to be updated since we return two things now. Can you rewrite that comment and make it accurate?